### DISCUSSION TITLE
Prayers replace animations!!!

### DISCUSSION THREAD

**Discussion by claytray** [runelite/runelite#17079] [Ideas]
Prayers replace animations.

Whatever attacks you could instead display the correct prayer to use instead. 

This would be helpful to learn what to pray to avoid getting damaged to help you get better at the game. (listen to sounds) 

Say you attack a dragon from range
Or even an aggressive monster is coming into Melle you, the animation of the attack it's going to use is replaced to the prayer you should use. When it does its attack the prayer you should use drops until it goes to attack you again. 

So instead of a fireball coming at you, it's the prayer you need to be using.

**Comment by DominickCobb-rs** [comment]
I believe this is against the [statement](https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1) about what is allowed in third party clients. Specifically "Prayer switching indicators".
